drivers/usb/usbaudio/pin.c

```diff
@@ -808,7 +808,7 @@ USBAudioPinCreate(
     }
 
     /* choose correct dataformat */
-    FormatIndex = GetDataRangeIndexForFormat(Pin->ConnectionFormat, Pin->Descriptor->PinDescriptor.DataRanges, Pin->Descriptor->PinDescriptor.DataRangesCount);
+    FormatIndex = GetDataRangeIndexForFormat(Pin->ConnectionFormat, (PKSDATARANGE *) Pin->Descriptor->PinDescriptor.DataRanges, Pin->Descriptor->PinDescriptor.DataRangesCount);
```

Comment:
Try to instead add the missing `const` to the parameter definition for the `GetDataRangeIndexForFormat` function.

Reply:
Works for me. Compilation still succeeds. Thanks.

Reply:
Fixed.